setup.py

```diff
@@ -8,7 +8,7 @@
       author_email="achamberlai9@bloomberg.net, fphillips7@bloomberg.net, dkiss1@bloomberg.net, dbeer1@bloomberg.net",
       license='Apache 2.0',
       packages=["github_webhook"],
-      install_requires=['flask', 'six'],
+      install_requires=['flask', 'six', 'enum34'],
```

Comment:
Perhaps use the `python_version` specifier here to limit this dependency to Python 2 only?